src/bo4e_generator/parser.py

```diff
@@ -163,34 +176,49 @@ def bo4e_init_file_content(namespace: dict[str, SchemaMetadata], version: str) -
     return init_file_content
 
 
-def remove_future_import(python_code: str) -> str:
+def remove_future_import(python_code: str, output_type: OutputType) -> str:
     """
     Remove the future import from the generated code.
+    If sql_model adapt SQLModel specific imports
     """
+    if output_type is OutputType.SQL_MODEL.name:
+        python_code = re.sub(r"from pydantic import (.*?)Field(.*?)\n", r"from pydantic import \1\2\n", python_code)
```

Comment:
Pack deine Regex-Pattern gerne in eine eigene Funktion oder ändere den Funktionsnamen in etwas passenderes. So suggeriert der Name ja sonst was anderes, als es dann tatsächlich tut ;)

Reply:
cca618ea8c732aff577cedb188102d7ef66733f3